### PR TITLE
Merge updates into master

### PR DESCRIPTION
**Pull request by mayorbyrne**
Was just playing around and re-familiarizing myself with the library, doing some unit testing, and realized if you pass nothing into `.pass()` or `.fail()` there was a trailing `-` logged. This PR attempts to fix this. No worries if this is garbage and gets rejected.

### CODE REVIEW

**Comment by coreybutler** [comment]
I'm finally getting around to this. Is there really a trailing `-`? The `-` is part of the TAP spec, so I'm assuming use of `pass()` or `fail()` with no parameter is interpreting this as an empty string.

I think something should be done about this, but instead of eliminating the dash, perhaps it would be better to have a default message like `pass` or `fail`. This way the output is clear. If someone sees a generic pass/fail message, it should be an indicator they need to specify a more specific message (i.e. prompt them to add a parameter to `pass()` or `fail()`).

In other words, [this line](https://github.com/coreybutler/tappedout/blob/69bfa95bac067efe3a34d520511555dad2546037/lib/suite.js#L158) `pass (msg = '', directive = null) {` would change to `pass (msg = 'pass', directive = null) {` and the fail function would do the same thing.
